web/src/locales/tr.json

```
@@ -28,19 +28,130 @@
+      "multiUser": {
+        "title": "Sınırsız Katılımcı",
+        "description": "İstediğiniz kadar arkadaşınızla birlikte izleyin"
+      },
+      "secure": {
+        "title": "Güvenli ve Özel",
+        "description": "Kişisel verileriniz korunur, reklamsız deneyim"
+      }
+    },
+    "howItWorks": {
+      "title": "Nasıl Çalışır?",
+      "subtitle": "Üç basit adımda birlikte izleme deneyimi",
```

Biome (2.1.2)

[error] The key howItWorks was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)